span.go

```diff
@@ -58,3 +62,30 @@ type Span interface {
 	// TraceContext.
 	AddToGoContext(goCtx context.Context) (Span, context.Context)
 }
+
+// See Span.Log(). Every LogData instance should specify at least one of Event
+// and/or Payload.
+type LogData struct {
+	// The timestamp of the log record; if set to the default value (the unix
+	// epoch), implementations should use time.Now() implicitly.
+	Timestamp time.Time
```

Comment:
`time.Time` is the canonical way to represent a moment in time in Go. It actually offers nanosecond precision (which is pretty silly given that modern CPUs *don't* offer nanosecond precision!).

FYI: https://golang.org/pkg/time/#Time